docs/UserGuide.adoc

```diff
@@ -27,8 +26,8 @@ AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for mana
 Having any Java 8 version is not enough. +
 This app will not work with earlier versions of Java 8.
 +
-.  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
+.  Download the latest `TuitionCor.jar` link:{repoURL}/releases[here].
+.  Copy the file to the folder you want to use as the home folder for your client information.
```

Comment:
Missing features breakdown in your UserGuide. Please add the features you expect every version to have in the form:
E.g.:
In v 1.2:  xxxxxxxxxxxxxxxxx

In v 1.3: xxxxxxxxxxxxxxxxxx

In v 1.4: xxxxxxxxxxxxxxxxxxxx

In v 1.5: xxxxxxxxxxxxxxxxxxxxxx

Also remember to say which features are coming in v2.0 (so we have an idea of the ultimate goal for your app)
Coming in v2.0: xxxxxxxxxxx

Reply:
t/urgent l/north g/p1 s/math

Reply:
Will update